docs/INSTALL.md

```diff
@@ -47,10 +47,12 @@ sudo dnf install -y \
     sqlite-devel
 ```
 
-**macOS**:
+**macOS**: 
+
+Make sure to follow the printed instructions to add Qt 5 to the PATH after installation.
 
 ```bash
-brew install qt cmake
+brew install cmake qt@5
```

Comment:
switched these around so that the qt instructions will be the last thing printed in the console